airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -80,6 +84,70 @@ def list_tasks(self, cluster: str, launchType: str, desiredStatus: str, family:
         ...
 
 
+class ECSTaskLogFetcher(Thread):
+    """
+    Fetches Cloudwatch log events with specific interval as a thread
+    and sends the log events to the info channel of the provided logger.
+    """
+
+    def __init__(
+        self,
+        *,
+        aws_conn_id: Optional[str] = None,
```

Comment:
Nitpick request:   This should now default to `'aws-default'` but it has not been updated everywhere yet.  **IFF** you are going to be making any revisions, could you change this default?  If not, I am going to be going through these and standardizing that in the near future, so no biggie.

See: http://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html

Reply:
sure, will keep that in mind

Reply:
```suggestion
        aws_conn_id: Optional[str] = 'aws_default`,
```

Reply:
done